include/k4a/k4a.hpp

```diff
@@ -109,7 +109,7 @@ class image
         reset();
     }
 
-    /** Sets image to a shallow copy of other
+    /** Sets image to a shallow copy of the other
```

Comment:
>the other [](start = 40, length = 9)

I think either of these is correct:
"shallow copy of other" (treating "other" as a reference to the actual name of the parameter), or
"shallow copy of the other image" (treating "other" as an adjective describing the image)

No strong feeling on which one to go with, but this way reads like it's missing a word; ditto for line 438